content/_index.md

```diff
@@ -3,21 +3,39 @@ title: "Home"
 date: 2021-10-01T09:28:27-05:00
 draft: false
 ---
-# LaunchCode
-You just created a brand new curriculum! Feel free to create new chapters or segments however you normally would. You can also find useful links below that may help inspire.
-You pulled in three LaunchCode created github submodules:
-- `layouts`: provides slides shortcode and overwrites menu footer
-- `static`: provides launchcode logo, favicon, and theme variants
-- `archetypes`: provides segment archetype for new chapters
-Example usage of segment:
-```bash
-hugo new --kind segment chapter-name
-```
-Upon running the above command it will create a new chapter named `chapter-name`
-{{% notice warning %}}
-Replace the content of this entire file with the desired homepage contents of this course!
-{{% /notice %}}
-## Helpful Links
-[LaunchCode Technical Training Github](https://github.com/LaunchCodeTechnicalTraining)
-[Relearn Documentation](https://mcshelby.github.io/hugo-theme-relearn/)
-[Live link to Linux Curriculum](https://lctt-linux.netlify.app/)
+
+# C# Web Development
+
+## What is this Course
+
+This course is an introduction to object-oriented programming in C#. Students will then move on to building web applications with the industry-grade framework, ASP.NET Core MVC.
+
+## Why take this Course
+
+This course is an introduction to **C#** and the .NET framework. C# is a programming language developed by Microsoft and used in almost all of its products.
```

Comment:
Why is C# bolded here, but not other places?

Reply:
I fixed the repetition. C# is bolded because we always bold important terms the first time they are seen in the book, but not afterward.

Reply:
Thanks Sally for the explanation.